docs/msbuild/msbuild-items.md

```diff
@@ -200,6 +200,61 @@ For more information about wildcard characters, see [How to: Select the files to
 </Target>
 ```
 
+#### MatchOnMetadata attribute
+
+The `MatchOnMetadata` attribute is applicable only to `Remove` attributes that reference other items (e.g. `Remove="@(Compile);@(Content)"`) and instructs the Remove operation to match items based on the values of specified metadata names, instead of matching based on the item values.
```

Comment:
```suggestion
The `MatchOnMetadata` attribute is applicable only to `Remove` attributes that reference other items (for example, `Remove="@(Compile);@(Content)"`) and instructs the Remove operation to match items based on the values of specified metadata names, instead of matching based on the item values.
```

Reply:
Microsoft style for docs is to avoid Latin abbreviations like e.g.